upload/src/addons/SV/WarningImprovements/XF/Repository/Warning.php

```diff
@@ -64,18 +68,40 @@ public function getSvWarningReplaceables(UserEntity $user, WarningEntity $warnin
                 $replacables['{' . $key . '}'] = $value;
             }
             $params = $replacables;
+            $params['content_action_options'] = $contentActionOptions;
```

Comment:
`$params['content_action_options']` needs to be a string-like. `getSvWarningReplaceables` returns `string[]`, the `$forPhrase` variable simple means if the key is being wrapped with `{` and `}`